plugins/permission-node/src/ServerPermissionClient.ts

```diff
@@ -110,16 +110,28 @@ export class ServerPermissionClient implements PermissionsService {
 
   async #getRequestOptions(
     options?: PermissionsServiceRequestOptions,
-  ): Promise<{ token?: string } | undefined> {
+  ): Promise<{ token?: string; identifier?: string } | undefined> {
     if (options && 'credentials' in options) {
       if (this.#auth.isPrincipal(options.credentials, 'none')) {
-        return {};
+        return { identifier: 'none' };
       }
 
-      return this.#auth.getPluginRequestToken({
+      const { token } = await this.#auth.getPluginRequestToken({
         onBehalfOf: options.credentials,
         targetPluginId: 'permission',
       });
+
+      let identifier: string | undefined = undefined;
```

Comment:
It's interesting that we've seen a couple of times now that people stringify credentials. We might want to standardise this.

Reply:
Yes, it would be nice to have common `toString` method in the `BackstageCredentials`